From fc39d58d9ab26f4cffec507ca0d1a0933c0f1590 Mon Sep 17 00:00:00 2001
From: Kathiravan T <quic_kathirav@quicinc.com>
Date: Wed, 19 Apr 2023 21:11:16 +0530
Subject: [PATCH 337/500] PCI: qcom: Add IPQ5332 PCIe support

IPQ5332 SoC comes with 3 Gen3 PCIe controllers. Init sequence is
similar to the controllers found on IPQ9574 SoC, so lets re-use the same
ops for IPQ5332 as well.

Change-Id: I31d524877cacec7d5e1b20bb440d2649af7b053e
Signed-off-by: Kathiravan T <quic_kathirav@quicinc.com>
---
 drivers/pci/controller/dwc/pcie-qcom.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/pci/controller/dwc/pcie-qcom.c b/drivers/pci/controller/dwc/pcie-qcom.c
index f963d461d1ee..331ca623f4ff 100644
--- a/drivers/pci/controller/dwc/pcie-qcom.c
+++ b/drivers/pci/controller/dwc/pcie-qcom.c
@@ -1623,6 +1623,7 @@ static const struct of_device_id qcom_pcie_match[] = {
 	{ .compatible = "qcom,pcie-apq8064", .data = &cfg_2_1_0 },
 	{ .compatible = "qcom,pcie-apq8084", .data = &cfg_1_0_0 },
 	{ .compatible = "qcom,pcie-ipq4019", .data = &cfg_2_4_0 },
+	{ .compatible = "qcom,pcie-ipq5332", .data = &cfg_1_27_0 },
 	{ .compatible = "qcom,pcie-ipq6018", .data = &cfg_2_9_0 },
 	{ .compatible = "qcom,pcie-ipq8064", .data = &cfg_2_1_0 },
 	{ .compatible = "qcom,pcie-ipq8064-v2", .data = &cfg_2_1_0 },
-- 
2.34.1

